.github/workflows/assets.yml

```diff
@@ -23,11 +23,14 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
+      - uses: pnpm/action-setup@v2
+        with:
+          version: '7.18.2'
       - run: git submodule absorbgitdirs
       - uses: actions/setup-node@v3
         with:
           node-version: '19'
-          cache: yarn
+          cache: pnpm
       - uses: actions/checkout@v3
         with:
           repository: lichess-org/ab
```

Comment:
This is no longer working, and I can't figure out why atm: https://github.com/lichess-org/lila/actions/runs/3724435275/jobs/6316587960#step:6:54

Reply:
nvm it's normal for a PR build